app/src/main/java/org/simple/clinic/contactpatient/views/CallPatientView.kt

```diff
@@ -7,7 +7,9 @@ import android.view.View
 import android.widget.Toast
```

Comment:
This feels like a rename rather than a new implementation. The earlier commit seems to be the newly added file.